deploy-to-aks/action.yml

```diff
@@ -0,0 +1,147 @@
+description: Deploy a docker image to an AKS environment
+
+inputs:
+  azure-credentials:
+    required: true
+  gcp-wip:
+    required: false
+  environment:
+    description: Environment to deploy to
+    required: true
+  github-token:
+    description: Default Github token retrieved via secrets.GITHUB_TOKEN or PAT with permission to the repository (Required)
+    required: true
+  pr-number:
+    description: PR number for the review app
+    required: false
+  sha:
+    description: Commit sha to be deployed
+    required: true
+  slack-webhook:
+    required: false
+  db-seed:
+    description: run seed command after a deployment
+    type: boolean
+    required: false
+    default: false
+  smoke-test:
+    description: Enable smoke test after deployment
+    type: boolean
+    required: false
+    default: false
+  healthcheck:
+    description: Health check path
+    required: false
+  terraform-base:
+    description: path to the terraform files
+    required: false
+    default: 'terraform/application'
+
+outputs:
+  environment_url:
+    description: The first external URL for the deployed environment
+    value: ${{ steps.set_output.outputs.DEPLOY_URL }}
+
+runs:
+  using: composite
+  steps:
+    - name: Set Environment variables
+      id: set_env_var
+      shell: bash
+      run: |
+        terraform_version=$(awk '/{/{f=/^terraform/;next}f' ${{ inputs.terraform-base }}/terraform.tf | grep -o [0-9\.]*)
+        echo "TERRAFORM_VERSION=$terraform_version" >> $GITHUB_ENV
+
+    - name: Use Terraform ${{ env.TERRAFORM_VERSION }}
+      uses: hashicorp/setup-terraform@v3
+      with:
+        terraform_version: ${{ env.TERRAFORM_VERSION }}
+        terraform_wrapper: false
+
+    - uses: azure/login@v2
+      if: ${{ inputs.db-seed == 'true' }}
+      with:
+        creds: ${{ inputs.azure-credentials }}
+
+    - uses: google-github-actions/auth@v2
+      if: ${{ inputs.gcp-wip != '' }}
+      with:
+        workload_identity_provider: ${{ inputs.gcp-wip }}
+
+    - uses: DFE-Digital/github-actions/set-kubelogin-environment@master
+      with:
+        azure-credentials: ${{ inputs.azure-credentials }}
+
+    - name: Terraform apply
+      shell: bash
+      run: make ${{ inputs.environment }} ci terraform-apply
+      env:
+        IMAGE_TAG: ${{ inputs.sha }}
+        PR_NUMBER: ${{ inputs.pr-number }}
+
+    - name: Install kubectl
+      if: ${{ ( inputs.db-seed == 'true' && inputs.environment == 'review' ) }}
+      uses: DFE-Digital/github-actions/set-kubectl@master
+
+    - name: Seed review app
+      shell: bash
+      if: ${{ inputs.db-seed == 'true' }}
+      run: make ci ${{ inputs.environment }} db-seed
+      env:
+        PR_NUMBER: ${{ inputs.pr-number }}
+
+    - name: set DEPLOY_URL output
+      shell: bash
+      id: set_output
+      run: |
+        first_url=$(terraform -chdir=${{ inputs.terraform-base }} output -json external_urls | jq -r '.[0]')
+        echo "DEPLOY_URL=${first_url}" >> $GITHUB_OUTPUT
```

Comment:
Can we rename to ENVIRONMENT_URL for consistency with the output?

Reply:
done